### PR TITLE
Sync Labels across most repos

### PR DESCRIPTION
**Pull request by t1mmen**
### In this PR:
* A central Github Labels configuration, to easily synchronize labels & usage across all relevant repositories.

### Why?
* Every repository will share (most) tags -- adios inconsistencies!
* Enables label sync with Productboard (better filtered views & reports)
* Identical tag naming across repositories makes automation much easier (e.g, Github Projects, Github Actions, Company Dashboard)
* Super-easy to add new tags, or change colors/description (renaming tags is harder)

### What this means for you:
* Labels not configured through Labelsync will be deleted on merge to `#master` (this setting _can_ be changed, but would prefer we don't)
* When you want to add new labels to a repo you control, please do so through `labelsync.ts` (see README for details)
* Github Projects using `project-bot` may need tweaking (I've updated https://github.com/orgs/Timely/projects/1 already)

Universal labels also makes the elusive "development dashboard" possible, and with much greater details (not pictured). A sneak-peak of the working prototype:

Screenshot 2020-04-21 09 48 10

### What I need from you today:
* Find repositories that you're involved in (comment below)
* Do you object to any new or renamed labels? Comment with your thoughts in the next 24 hours!
* Are any of the labels slated for removal important to you? Why? (NB! Most used labels exist with a slightly different name, so double check)
* Do you see a need for tags we're missing, and if so, for which repos? All of them?

### Next Steps:
* @t1mmen will rename existing tags that have a new name, to avoid having to re-tag everything.

### What it looks like in practice
* All labels @ https://github.com/Timely/project-management/labels
* More realistic usage on Frontend @ https://github.com/Timely/Frontend/labels

### CODE REVIEW

**Comment by pierrickatkyoto** [comment]
All AI labels can be changed, full agreement.

File: labelsync.yml [hunk 1]
```diff
@@ -604,6 +619,217 @@ repos:
...
+        siblings: []
+      'Colab: Help Wanted':
+        color: '#7057ff'
+        alias: []
+        siblings: []
+      'Colab: Early Feedback Wanted':
+        color: '#7057ff'
+        alias: []
+        siblings: []
+      'QA: Ready for review':
```
Comment (truptimemoryai) [inline labelsync.yml:720]:
```suggestion
      'QA: Ready for QA':
```

Comment (truptimemoryai) [reply labelsync.yml:720]: cc @amelija

File: labelsync.yml [hunk 2]
```diff
@@ -764,3 +990,2443 @@ repos:
         color: '#8042dd'
         alias: []
         siblings: []
+  timely-android:
```
Comment (truptimemoryai) [inline labelsync.yml:993]: Also out of curiosity, can we use 'anchors' to maintain the labels which are same across all repositories and then just append the ones which are different?
http://blogs.perl.org/users/tinita/2019/05/reusing-data-with-yaml-anchors-aliases-and-merge-keys.html

Comment (truptimemoryai) [reply labelsync.yml:993]: Ok seems like this is added by some bot. So you can safely ignore this comment.

File: labelsync.yml (hunk 1 above)
Comment (t1mmen) [reply labelsync.yml:720]: @truptimemoryai 

I was hoping to keep the format of `[Team/Role]: State of PR/issue` consistent, so we'd have:

* QA: Ready for review
* Dev: Ready for review
* Design: Ready for review
* UI Copy: Ready for review

... and whatever else we come up with.

"QA: Ready for QA" reads a bit redundant to me, any particular reason you'd prefer that naming?

File: labelsync.yml (hunk 2 above)
Comment (t1mmen) [reply labelsync.yml:993]: That's pretty much what `labelsync.ts` already does (the yaml parser of labelsync isn't 100% solid, so it was easier to do this via JS/TS)

File: labelsync.yml (hunk 1 above)
Comment (truptimemoryai) [reply labelsync.yml:720]: @t1mmen 
The labels are similar to dev reviews and also show up. Atleast can we have a different color to differentiate with Dev and QA review?

**Pull request merged**